modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.soy

```diff
@@ -8,7 +8,7 @@
  * @param changeableDefaultLanguage
  * @param defaultLocale
  * @param? elementClasses
- * @param? id
+ * @param id
  * @param locales
```

Comment:
@fernandosouza Can you define the parameters using the new syntax? See https://github.com/liferay/com-liferay-osb-faro-build-private/wiki/Metal-Soy-Guidelines#declaring-params